### PR TITLE
115 update figshare

### LINKED ISSUE #115: Update figshare
https://figshare.com/articles/dataset/Geospatial_Python_Workshop_Data/25721754

- [x] All bands after fire
- [x] DEM
- [x] Administration boundaries
- [x] OSM data

after uploading:

- [x] Update setup.md

### PR DESCRIPTION
**Pull request by rogerkuou**
This should be merged after PR #112.

fix #115

@Morrizzzzz I have uploaded data on Figshare and updated the setup instruction. Please review and merge PR #112 first, since this PR also uses part of changes in PR 112.

Data updated in Figshare: https://figshare.com/ndownloader/articles/25721754

BTW, I also attached the notebook used to download image: [ep5_prepare_data.zip](https://github.com/esciencecenter-digital-skills/geospatial-python/files/15190049/ep5_prepare_data.zip), in case you want to check how the images are created.

### CODE REVIEW

**Comment by rogerkuou** [comment]
Hi @Morrizzzzz, I updated to the figshare link to: https://figshare.com/ndownloader/articles/25721754/versions/3. This one is with a version number and it works.

File: episodes/08-crop-raster-data.md
Comment (Morrizzzzz) [inline episodes/08-crop-raster-data.md:1]: Has been reviewed in PK 110

File: episodes/07-vector-data-in-python.md
Comment (Morrizzzzz) [inline episodes/07-vector-data-in-python.md:1]: @rogerkuou you reviewed this in issue 92, right?

File: episodes/fig/E07/rhodes_assets.png
Comment (Morrizzzzz) [inline episodes/fig/E07/rhodes_assets.png:1]: to improve the readability of the map, maybe put the legend down and use more distinct colors for the legend items.

**Review by Morrizzzzz — COMMENTED** [review]
minor comments

**Pull request merged**